src/containers/EditMarkupPage/EditMarkupPage.tsx

```diff
@@ -129,7 +129,7 @@ interface MarkupLocationState {
   backUrl?: string;
 }
 
-interface Props extends RouteComponentProps<MatchParams, StaticContext, MarkupLocationState> {}
+interface Props extends RouteComponentProps<MatchParams, {}, MarkupLocationState> {}
```

Comment:
RouteComponent tar StaticContext som variabel. Siden StaticContext ikke brukes, ønsket jeg å ikke sende med StaticContext. Tenkte da {} var bedre enn å skulle importere noe ubrukt. Prøvde meg med Omit<RouteComponentProps, 'StaticContext'> for å ignorere feltet, men fant ikke en løsning (se https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys)